app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="sessions-page">
+
```

Comment:
- [ ] OPTIONAL
Your app is missing a splash screen. Kindly ensure to add a simple page with the name of your app. This could have a transition to the login/ sign up page or with links to the sign up/log in page